app/serializers/api/v2/serializable_course.rb

```diff
@@ -16,7 +16,8 @@ def enrichment_attribute(name, enrichment_name = name)
                  :content_status, :ucas_status, :funding,
                  :level, :is_send?, :has_bursary?, :has_scholarship_and_bursary?,
                  :has_early_career_payments?, :bursary_amount, :scholarship_amount,
-                 :english, :maths, :science, :gcse_subjects_required, :age_range_in_years
+                 :english, :maths, :science, :gcse_subjects_required, :age_range_in_years,
+                 :accrediting_provider, :accrediting_provider_code
```

Comment:
`:accredited_body, :accredited_body_code`?

Reply:
I've just used the terms as seen in the model, do you propose we rename them?